tests/acceptance/pageObjects/sharedWithMePage.js

```diff
@@ -160,6 +181,10 @@ module.exports = {
     },
     batchDeclineSharesButton: {
       selector: '.oc-files-actions-decline-share-trigger'
+    },
+    batchButton: {
+      selector: '//button[contains(@class,"oc-button") and span[text()="%s"]]',
```

Comment:
I won't block this, but just FYI: I have a strong opinion that using strings as selectors makes tests cumbersome. Whenever you change text in the code or start to take translations into account in e2e testing the test will break and would need to be adapted. In my opinion tests are more resilient when you use more stable selectors.
Again, keep it like it is, I think it's not worth the effort to change it now. 😉